code/datums/components/COMPONENT_TEMPLATE.md

```diff
@@ -16,12 +16,17 @@ See _component.dm for detailed explanations
 		send_to_playing_players(myargtwo)
 
 /datum/component/mycomponent/RegisterWithParent()
-	RegisterSignal(parent, COMSIG_NOT_REAL, ./proc/signalproc)                                    // RegisterSignal can take a signal name by itself,
-	RegisterSignal(parent, list(COMSIG_NOT_REAL_EITHER, COMSIG_ALMOST_REAL), ./proc/otherproc)    // or a list of them to assign to the same proc
+	RegisterSignal(parent, COMSIG_NOT_REAL, PROC_REF(signalproc))
+		// RegisterSignal can take a signal name by itself,
+	RegisterSignals(parent, list(COMSIG_NOT_REAL_EITHER, COMSIG_ALMOST_REAL), PROC_REF(otherproc))
+		// or a list of them to assign to the same proc
+		//! if signals are a list, use 'RegisterSignals' with extra s.
+		//! if it's a single signal, use 'RegisterSignal' without s
 
 /datum/component/mycomponent/UnregisterFromParent()
 	UnregisterSignal(parent, COMSIG_NOT_REAL)          // UnregisterSignal has similar behavior
-	UnregisterSignal(parent, list(                     // But you can just include all registered signals in one call
+	UnregisterSignal(parent, list(
+			/* But you can just include all registered signals in one call */
```

Comment:
Even though the comments dont make sense as actual code, the old formatting was better since it was consistent and lined up